src/main/java/edu/northeastern/truthtree/AppConst.java

```diff
@@ -4,23 +4,51 @@
  * Represents the constants used throughout the application.
  */
 public class AppConst {
-	// App Constants
```

Comment:
Consider using 4 indents in this file as others in the project

Reply:
I can change the indentation, but my settings are already set to indent size: 4. It looks like the previous is about twice that so 8?

Reply:
I have committed the indentation change and can confirm that the AppConst was at 8 for the indentation previously.